react/components/SideBar.tsx

```diff
@@ -19,9 +18,9 @@ interface Chapter {
 }
 
 const SideBar: FunctionComponent = () => {
-  const { appName } = useAppNameAndFile()
+  const appName = useAppNameAndFile().appName || 'vtex.io-documentation@0.x'
```

Comment:
`useAppNameAndFile()` will always return an object? if it returns nil, this would break

Reply:
Yes! The return statement for `useAppNameAndFile()` is:
`return { appName: finalAppName, fileName }`, where `finalAppName` is either a string or `undefined`, and `fileName` is always a string.